SmartOrder.csproj

```diff
@@ -74,13 +81,24 @@
       WorkingDirectory="$(MSBuildProjectDirectory)" 
     />
   </Target>
-
+  
+  <!-- Normal build process -->
   <Target
-    Name="Build TailwindCSS" 
-    AfterTargets="Build"
+    Name="Build TailwindCss"
+    BeforeTargets="PreBuildEvent"
+    DependsOnTargets="FindTailwindCss"
+    Condition="'$(DotNetWatchBuild)' != 'true'"
   >
-    <Exec 
-      Command="$(ProjectDir)build-tailwind.sh" 
-    />
+    <Exec Command="$(TailwindCssPath) build -i ./Styles/site.css -o ./wwwroot/css/site.css" />

```

Comment:
To a local what?

Reply:
One of the things that would be really great with the custom tasks approach: we could have MSBuild gather the set of files together using the `<ItemGroup>`, and pass the entire list to the custom task.

Essentially:
* MSBuild scans the project subdirectories for styles
  * In watch mode, we can make it watch for changes in those particular files, and only reprocess those ones
* Give the custom task the entire list of files, as well as an output directory
* The custom task takes charge of looping and feeding them to tailwind
* Tailwind then processes them down into their final state, dumping them into the defined output directory.

The way this is currently set up, there's not a very good way to do that, and definitely not one that's cross-platform.

Reply:
We can hard code the style paths for the `-i` and `-o` parameters into a property group for now; I don't see an issue with that.  Later on, we might want to make it more dynamic, so that it uses the `<ItemGroup>` with its wildcard method to find the needed input files, and use those files to determine the proper outputs.

I can look through the builtin tasks and see if there's anything that lets us loop over a set of arguments, and pass each set to a shell process.  I don't think there is, though.

Reply:
Hardcoding as property group for now sounds good.